tests/transports-governance/conftest.py

```
@@ -0,0 +1,668 @@
+
+
+# Utility functions
+
+
+def generate_unique_name(prefix: str = "Test") -> str:
+    """Generate a unique name for testing"""
+    return f"{prefix} {uuid.uuid4().hex[:8]} {int(time.time())}"
+
+
+def wait_for_condition(
+    condition_func, timeout: float = 5.0, interval: float = 0.1
```

Ruff (0.12.2)

Missing type annotation for function argument `condition_func`

(ANN001)